stacklogs-looker/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
LICENSE file is already on the top level folder. We probably don't have to put it under this folder.